frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -0,0 +1,381 @@
+    cy.get(".Header-buttonSection").click("left");
+    cy.findByText("Orders Over Time").click();
+
+    cy.contains("Order Totals by State");
+    cy.contains("Orders Over Time");
+
+    cy.findByText("Save").click();
+
+    // ================
+    // should add a new user
+    // =================
+
+    // Sets up route
+    cy.server();
+    cy.route({
+      method: "POST",
+      url: "/api/user",
+    }).as("createUser");
+
+    // Navigates through admin pages
+    cy.get(".Nav")
+      .children()
+      .last()
+      .children()
+      .last()
+      .click();
+    cy.findByText("Admin").click();
+
+    cy.contains("Metabase Admin");
+    cy.contains("dashboard").should("not.exist");
+
+    cy.findByText("People").click();
+
+    cy.contains("Groups");
+
+    // Inputs user info (first modal)
+    cy.findByText("Add someone").click();
+    cy.findByLabelText("First name").type(new_user.first_name);
+    cy.findByLabelText("Last name").type(new_user.last_name);
+    cy.findByLabelText("Email").type(new_user.username);
+    cy.get(".ModalBody")
+      .find(".Icon-chevrondown")
+      .click();
+    cy.findByText("English").click();
+    cy.findByText("Create").click();
+
+    cy.wait("@createUser").then(xhr => {
+      cy.wrap(xhr.request.body.password).as("password");
+    });
+
+    //  Password confirmation(second modal)
+    cy.contains("has been added");
+    cy.findByText("Done").click();
+
+    cy.contains(new_user.username);
+
+    // ****************
+    // *** NEW USER ***
+    // *****************
+
+    // =================
+    // New user can sign in
+    // =================
+
+    signOut();
+    cy.get("@password").then(pass => {
+      cy.visit("/");
+      cy.findByLabelText("Email address").type(new_user.username);
+      cy.findByLabelText("Password").type(pass);
+      cy.findByText("Sign in").click();
+      cy.contains(new_user.first_name);
+
+      // =================
+      // should see questions currently in the "Our Analytics" collection as user
+      // =================
+
+      cy.findByText("Browse all items").click();
+      cy.contains("My personal collection");
+
+      // =================
+      // should see dashboard in the "Our Analytics" collection as user
+      // =================
+
+      cy.findByText("Dashboards").click();
+      cy.contains("Demo Dash 2");
+
+      // =================
+      // should create my own question as user
+      // =================
+
+      cy.findByText("Ask a question").click();
+
+      cy.contains("Native query");
+
+      cy.findByText("Simple question").click();
+      cy.findByText("Sample Dataset").click();
+      cy.findByText("Reviews").click();
+
+      cy.get(".Button")
+        .contains("Summarize")
+        .click();
+      cy.get(".scroll-y")
+        .contains("Rating", { timeout: 20000 })
+        .click();
+      cy.findByText("Done").click();
+
+      cy.contains("Auto binned");
+      cy.get(".Icon-bar");
```

Comment:
Let's go ahead and save this as a user as well. Try to do it in the user's personal collection - I know we had issue with the admin collection, but for the user it should work.

Reply:
The user is also unable to save into their personal collection (same thing happens as admin: blank input and "Referential integrity constraint violation:"
 error when you click Save)

<img width="1111" alt="Screen Shot 2020-06-11 at 8 51 07 AM" src="https://user-images.githubusercontent.com/58329466/84408190-b3fd1680-abc0-11ea-9331-ca1732dd6236.png">